src/truncate.js

```diff
@@ -27,10 +27,22 @@ function truncate(maxLength, value) {
   return when(
     isNotNilOrEmpty,
     compose(
-      stringToTruncate =>
-        `${take(maxLength - ELLIPSIS.length, stringToTruncate)}${
-          stringToTruncate.length > maxLength ? ELLIPSIS : ''
-        }`,
+      stringToTruncate => {
```

Comment:
This chain looks a bit complicated to _mantain_ due to the nested conditions.

Reply:
Makes sense. What about now?